setup.py

```diff
@@ -30,25 +31,28 @@ def local_open(fname):
 for directory in os.walk('example/templates'):
     example_dirs.append(directory[0][8:] + '/*.*')
 
+with local_open(os.path.join('pages', '__init__.py')) as f:
+    exec(f.read(), {}, pages)
+
 url_schema = 'http://pypi.python.org/packages/source/d/%s/%s-%s.tar.gz'
-download_url = url_schema % (package_name, package_name, pages.__version__)
+download_url = url_schema % (package_name, package_name, pages['__version__'])
 
 install_requires = [
-    'django-mptt>=0.8.3,<0.9.0',
-    'six>=1.10.0,<1.11.0',
-    'Pillow>=3.2.0,<3.3.0',
-    'tqdm>=4.4.0,<4.5.0',
-    'django-taggit>=0.18.1,<0.19.0',
-    'requests>=2.9.0,<3.0.0',
+    'django-mptt>=0.8.3',
+    'six>=1.10.0',
+    'Pillow>=3.2.0',
+    'tqdm>=4.4.0',
+    'django-taggit>=0.18.1',
+    'requests>=2.9.0',
```

Comment:
if the tests are ran once in a while, you should be able to catch that. But hey, who knows, things will break this is life.

The `requirements.txt` file is important for the people using `django-page-cms`. E.g. you do not provide all the combination of setups, e.g. `pages + haystack + ckeditor` but no `markdown`. Someone else is responsible to pick the libraries she needs and fix a concret version that you do support, e.g.
```
Django==1.9.8
django-page-cms==1.9.4
Python-Markdown==2.6.6
...
```